pay-api/src/pay_api/utils/converter.py

```diff
@@ -1,25 +1,76 @@
 """Converter module to support decimal and datetime serialization."""
-from decimal import Decimal
+import re
 from datetime import datetime
+from decimal import Decimal
+from enum import Enum
 from typing import Any, Dict, Type
 import cattrs
+from attrs import fields, has
+from cattrs.gen import make_dict_structure_fn, make_dict_unstructure_fn, override
 
 
 class Converter(cattrs.Converter):
     """Addon to cattr converter."""
 
-    def __init__(self):
+    def __init__(self, camel_to_snake_case: bool = False, enum_to_value: bool = False):
```

Comment:
We should probably convert this into a data class for ConverterOptions if we start adding more configurations in the future